tox.ini

```diff
@@ -43,11 +44,11 @@ setenv =
 
 
 [testenv:black]
-commands = black --skip-string-normalization --line-length=100 src/ tests/
+commands = black --skip-string-normalization --line-length=99 src/ tests/
 deps = black
 
 [testenv:lint]
-commands = flake8 src/ tests/
+commands = flake8 --ignore=W503,C901 src/ tests/
```

Comment:
Would it be possible to ignore this for specific lines of code as in https://flake8.pycqa.org/en/3.1.1/user/ignoring-errors.html#in-line-ignoring-errors?

Reply:
I use the inline ignore to ignore C901: the function complexity warning. In the case of W503, due to a bug in flake, W503 conflicts with W504, so one of the two must be disabled completely.

Reply:
Understood. Thnx